src/gui/org/deidentifier/arx/gui/view/impl/utility/ViewClassificationAttributes.java

```diff
@@ -196,33 +207,32 @@ private void update() {
 
         root.setRedraw(false);
         
-        Set<String> selectedFeatures = model.getSelectedFeatures();
-        
         for (TableItem item : features.getItems()) {
             item.dispose();
         }
-        
-        for (int i = 0; i < handle.getNumColumns(); i++) {
-            TableItem item = new TableItem(features, SWT.NONE);
-            String value = handle.getAttributeName(i);
-            item.setText(value);
-            item.setChecked(selectedFeatures.contains(value));
-        }
-        
-        Set<String> selectedClasses = model.getSelectedClasses();
-        
         for (TableItem item : classes.getItems()) {
             item.dispose();
         }
         
-        for (int i = 0; i < handle.getNumColumns(); i++) {
-            TableItem item = new TableItem(classes, SWT.NONE);
-            String value = handle.getAttributeName(i);
-            item.setText(value);
-            item.setChecked(selectedClasses.contains(value));
+        for (int col = 0; col < handle.getNumColumns(); col++) {
+            String attribute = handle.getAttributeName(col);
+            DataDefinition def = model.getOutputDefinition() == null ? model.getInputDefinition() : model.getInputDefinition();

```

Comment:
Should be:

DataDefinition def = model.getOutputDefinition() == null ? model.getInputDefinition() : model.getOutputDefinition();